package.json

```diff
@@ -1,51 +1,105 @@
 {
-  "name": "remix-app-template",
-  "private": true,
-  "description": "",
-  "license": "",
-  "sideEffects": false,
+  "name": "git-truck",
+  "version": "0.5.15",
+  "private": false,
+  "description": "Visualizing a Git repository",
+  "license": "MIT",
+  "main": "./build/index.js",
+  "bin": "./build/index.js",
   "scripts": {
-    "build": "npx cross-env NODE_ENV=production remix build",
-    "dev": "npm run parse --",
-    "postdev": "npx cross-env NODE_ENV=development remix dev",
-    "postinstall": "remix setup node",
-    "start": "npm run parse --",
-    "poststart": "npx cross-env NODE_ENV=production remix-serve build",
-    "clean": "npx rimraf -rf ./parser/dist data.json build",
-    "parse": "npm --prefix ./parser/ install && npm --prefix ./parser/ run build && npm --prefix ./parser/ start -- --gittruckroot .. --path . --out ./data.json"
+    "test": "jest --coverage",
+    "clean": "npx rimraf -rf ./parser/dist build public/build",
+    "build": "remix setup node && cross-env NODE_ENV=production remix build && node ./post-build.js",
+    "dev": "cross-env NODE_ENV=development remix build && run-p dev:*",
+    "dev:remix": "cross-env NODE_ENV=development remix watch",
+    "dev:node": "cross-env NODE_ENV=development nodemon ./build/index.js --watch ./build/index.js",
+    "start": "cross-env NODE_ENV=production node ./build/index.js"
```

Comment:
Is it possible to have a command for just running the parser, or just running the app?

Reply:
Yeah, that makes sense, but I would think it is out of scope of this monster PR, no?